system/Test/FeatureResponse.php

```diff
@@ -98,6 +98,27 @@ public function assertRedirect()
 		$this->assertTrue($this->isRedirect(), 'Response is not a redirect or RedirectResponse.');
 	}
 
+	/**
+	 * Assert that a given response was a redirect
+	 * and it was redirect to a specific URI.
+	 *
+	 * @param string $uri
+	 */
+	public function assertRedirectedTo(string $uri)
```

Comment:
> and it was redirect

> assertRedirectedTo

I think we should choose a case and stick with it. I would prefer "redirect" to be consistent with the other methods (so `assertRedirectTo()`) but as long as it is consistent and clear.